src/main/java/com/squareup/javapoet/ClassName.java

```diff
@@ -122,7 +122,7 @@ public static ClassName bestGuess(String classNameString) {
 
     // Add the package name, like "java.util.concurrent", or "" for no package.
     int p = 0;
-    while (p < classNameString.length() && Util.isLowerCase(classNameString.charAt(p))) {
+    while (p < classNameString.length() && Character.isLowerCase(classNameString.charAt(p))) {
```

Comment:
Definitely.

On Tue Jan 27 2015 at 8:31:37 PM Éamonn McManus <notifications@github.com>
wrote:

> In src/main/java/com/squareup/javapoet/ClassName.java
> <https://github.com/square/javapoet/pull/208#discussion_r23630264>:
>
> > @@ -122,7 +122,7 @@ public static ClassName bestGuess(String classNameString) {
> >
> >      // Add the package name, like "java.util.concurrent", or "" for no package.
> >      int p = 0;
> > -    while (p < classNameString.length() && Util.isLowerCase(classNameString.charAt(p))) {
> > +    while (p < classNameString.length() && Character.isLowerCase(classNameString.charAt(p))) {
>
> Fair enough. It's not a reason not to do things properly, though. Just
> because people *might* have problems if they use such identifiers doesn't
> mean that we can just throw them under the bus here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/square/javapoet/pull/208/files#r23630264>.
>